### PR TITLE
Contribute umbilici for scroll 1, 2 and 4.

### PR DESCRIPTION
**Pull request by spelufo**
Hi. These are what I had traced in blender for Scroll 1 and Scroll 4 (PHerc 1667). I've translated them to your coordinate system.

They have one point at the middle z coordinate of each grid cell layer. That's why the z coordinate for the scroll 1 umbilicus increments by 500.

The scroll 4 one I had done on the 3.24um canonical scan. I've scaled it to fit the 7.91um scan, which I think should work because the scroll_data page says "All volumes will be aligned to the canonical volume.". If this is not the case each umbilicus needs to be traced separately and this one is wrong.

### CODE REVIEW

**Comment by spelufo** [comment]
I've just traced the core for scroll 2 in blender too. It was a bit hard to follow in some places, it really twists and turns.

**Comment by schillij95** [comment]
Let me look at my own umbilici for scroll 2 and 4, will get back to this in a few days. Not sure how i can incorporate different versions of umbilici for a scroll

**Comment by schillij95** [comment]
Close, decide to not host all the umbilici inside this repository.

Hard to track due to: different resolutions, scans, scrolls, uncertainty in how to evaluate accuracy of many umbilici.

Umbilici are easy and fast to generate when using thaumato with the GUI. ("Generate Umbilicus" button)
